docs/guides/ruler-high-availability.md

```diff
@@ -0,0 +1,45 @@
+---
+title: "Ruler High Availability"
+linkTitle: "Ruler high availability"
+weight: 10
+slug: ruler-high-availability
+---
+
+This guide explains the concepts behind ruler high availability and when to use this feature
+
+## Background
+
+When rulers are deployed using shuffle sharding, each rule group is evaluated by a single ruler only. All the rulers in
```

Comment:
Can we move the doc to its own PR?

Reply:
Yes. Sorry about this. This was supposed to be in its own PR